src/app/space/create/pipelines/pipelines.component.ts

```diff
@@ -116,21 +116,37 @@ export class PipelinesComponent implements OnInit, OnDestroy {
       }));
 
     this.subscriptions.push(
-      this.runtimePipelinesService.current.subscribe((buildConfigs: BuildConfig[]) => {
-        this._allPipelines = buildConfigs;
-        this.applyFilters();
-        this.applySort();
-      }));
+      this.runtimePipelinesService.current.combineLatest(
+        this.pipelinesService.getOpenshiftConsoleUrl().do((url: string) => {
+          if (url !== '') {
+            this.consoleAvailable = true;
+          } else {
+            this.consoleAvailable = false;
+          }
+          this.openshiftConsoleUrl = url;
+        }),
+        this.setupBuildConfigLinks)
+        .subscribe((buildConfigs: BuildConfig[]) => {
+          this._allPipelines = buildConfigs;
+          this.applyFilters();
+          this.applySort();
+        })
+    );
+  }
 
-    this.subscriptions.push(
-      this.pipelinesService.getOpenshiftConsoleUrl().subscribe((url: string) => {
-        if (url !== '') {
-          this.consoleAvailable = true;
-        } else {
-          this.consoleAvailable = false;
+  private setupBuildConfigLinks(buildConfigs: BuildConfig[], consoleUrl: string): BuildConfig[] {
+    if (consoleUrl && consoleUrl !== '') {
```

Comment:
Can it be the case that `consoleUrl` evaluates as truthy, but then it is also the empty string? I think this condition can just be `consoleUrl`, since the empty string is falsy.